<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupForm</class>
 <widget class="QDialog" name="SetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup Style</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>RPM</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>RGB value</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_FoldLength"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_1"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_1"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_2"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tension</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Region 2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Region 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Fold length</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Region 3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Region 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_3"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_2"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_3"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_4"/>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_4"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Save and Run</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_back">
     <property name="text">
      <string>Back to Main</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox_RGB"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
